src/redux/store.ts

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import loader from 'redux/reducers/loader';
+
+export const store = configureStore({
+  reducer: {
+    loader: loader.reducer,
```

Comment:
https://github.com/Programmers-Only-Group/frontend/pull/119/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r505665488 then you could only use `loader` as exported one